health-sdk/sdk/src/main/java/net/gini/android/health/sdk/review/ReviewViewModel.kt

```diff
@@ -50,6 +60,11 @@ internal class ReviewViewModel(val giniHealth: GiniHealth, val configuration: Re
     private val _paymentProviderApp = MutableStateFlow<PaymentProviderApp?>(null)
     val paymentProviderApp: StateFlow<PaymentProviderApp?> = _paymentProviderApp
 
+    private val _paymentNextStep = MutableSharedFlow<PaymentNextStep>(
+        extraBufferCapacity = 1,
+        onBufferOverflow = BufferOverflow.DROP_OLDEST)
```

Comment:
[Clarification] Can you help me please to understand why buffering is needed?

Reply:
The `extraBufferCapacity` is just so the flow emits events. If we remove it, then nothing is emitted. 
I've removed `onBufferOverflow` as it was not adding any value - originally I thought it was helping with events to not be collected again on configuration change, but it seems to be working without it.